tests/kernels/test_cutlass.py

```diff
@@ -190,3 +190,44 @@ def test_cutlass_subset():
                         b.to(dtype=torch.float32)).to(dtype=torch.bfloat16)
 
     assert torch.allclose(out, baseline, rtol=1e-1, atol=1e0)
+
+
+# Test to make sure eager mode works
+class CutlassLayer(torch.nn.Module):
+
+    def __init__(self, b, scale_a, scale_b, out_dtype):
+        super(CutlassLayer, self).__init__()
```

Comment:
I didn't think `super()` required any arguments?  e.g. I thought you can just do `super().__init()__`